### ISSUE TITLE
Overriding Firefox 24.0 black background in fullscreen mode

### ISSUE THREAD

**Issue by scottdavis99** [hakimel/reveal.js#668]
I'm not sure if this is a bug or expected behavior in Firefox 24.0, but when you enter fullscreen mode by pressing the "f" key, the background color is set to black. (This makes black text on a black background invisible, although my images still appear.) I've replicated this on both Windows and OS X.

NOTE: this behavior DOESN'T manifest when I click the double-arrow key in the upper right corner on OS X -- only when I press the "f" key.

One workaround is to hardcode the bgcolor attribute on the body element:

``` html
<body bgcolor="white">
```

Another is to use the -moz-full-screen CSS selector with !important to override.

``` css
      body:-moz-full-screen {
        background-color: #fff !important;
      }
```

**Comment by DrWatsono** [comment]
Hey, I just encountered this behaviour in Chrome 45.0.2454 on Windows and OS X 10.10.4. In  This is also present on the reveal demo site:

![revealbug](https://cloud.githubusercontent.com/assets/3072741/10069248/2e91ea2e-62ab-11e5-82e2-6a7a696c5e4c.gif)

The fix suggested in this answer is working: http://stackoverflow.com/questions/16163089/background-element-goes-black-when-entering-fullscreen-with-html5#answer-24727000

**Comment by mplabs** [comment]
This seems to be a chrome bug. Adding html { background-color: inherit; } seems to help.

**Comment by DrWatsono** [comment]
That also works. Thanks, Felix!

**Comment by hendrikstill** [comment]
I have the same problem with Chrome 45.0.2454.99 (64-bit) on OSX 10.10.5 
The fix from @mplabs works for me, too.

**Comment by ingben** [comment]
:+1: @mplabs. Just saved me quite some time. Works with Chrome Version 45.0.2454.101 (64-bit) on OS X 10.11.

**Comment by CodeAtMeBro** [comment]
@mplabs solution works for me, but also overides my background gradients that I've set on body{background:...} Does anyone have a suggestion on how to maintain these into fullscreen display?

**Comment by hakimel** [comment]
I tested and was seeing the same issue in Chrome OS X. Just pushed up a fix in 5e763bc67d140d1a41056c2e63f0fbe80c5d03ab

Can someone who was experiencing this issue in Firefox give it another try against the latest code and let me know if it works there too?

**Comment by huijing** [comment]
I've tested the latest code and is confirmed working in firefox as well.
